src/state/apis/zapper/zapperApi.ts

```diff
@@ -36,7 +36,7 @@ const headers = {
   authorization,
 }
 
-const zapperClient = createApiClient(ZAPPER_BASE_URL)
+const zapperClient = createApiClient(ZAPPER_BASE_URL, { validate: 'none' })
```

Comment:
revert out of this PR - discussed with gomes specifically related to uni v2, replace `zod` with `myzod` for 25x improvement on 1mb of JSON validation